actions/lib/purge_remote_folder.py

```diff
@@ -0,0 +1,127 @@
+#!/usr/bin/python
+
+import argparse
+import datetime
+import json
+import logging
+import os
+import shutil
+import socket
+
+
+class PurgeRemoteFolders(object):
+
+    def __init__(self, biotank_archive_folder, age_in_days, dryrun):
+        self.biotank_archive_folder = str(biotank_archive_folder)
+        self.age_in_days = int(age_in_days)
+        self.dryrun = bool(dryrun)
+
+        hostname = socket.gethostname()
+        self.directory = self.sanitize_path(
```

Comment:
directory is a bit generic, could it be named `archive_base_dir` or similar?

Reply:
definitely, good point